client/components/domains/domain-suggestion/test/index.js

```diff
@@ -27,16 +30,44 @@ describe( 'Domain Suggestion', function() {
 	} );
 
 	describe( 'has attributes', () => {
-		it( 'should have data-e2e-domain attribute for e2e testing', () => {
-			const domainSuggestion = shallow( <DomainSuggestion
-				domain="example.com" isAdded={ false }/> );
-			if ( domainSuggestion.props()[ 'data-e2e-domain' ] ) {
-				expect( domainSuggestion.props()[ 'data-e2e-domain' ] ).to.equal( 'example.com' );
-			} else {
+		context( 'AB test clickableButton', () => {
```

Comment:
Personally I follow recommendations shared in Mocha documentation:

![screen shot 2016-09-26 at 12 26 34](https://cloud.githubusercontent.com/assets/699132/18831168/8ec395b8-83e4-11e6-85d4-6c36ca4ca03c.png)